models/sessions/scratch/snowplow_unified_sessions_this_run.sql

```diff
@@ -295,8 +295,9 @@ select
   , f.screen_resolution
 
 
-  {% if var('snowplow__enable_mobile_context') %}
+  {% if var('snowplow__enable_mobile_context')  %}
     {{ snowplow_unified.mobile_context_fields('f')}}
+    , coalesce(iso_639_2t_2_char.name, iso_639_2t_3_char.name, iso_639_3.name, mobile__language) as mobile_language_name
```

Comment:
i believe mobile__language is a column from the  session_firsts f table

if so could you change this to f.mobile__languages 

its a small annoying change it will help us in the future